source/LogbookPanel.cpp

```diff
@@ -243,7 +245,15 @@ bool LogbookPanel::Click(int x, int y, int clicks)
 
 bool LogbookPanel::Drag(double dx, double dy)
 {
-	scroll = max(0., min(maxScroll, scroll - dy));
+	if ((hoverPoint.X() - Screen::Left()) > SIDEBAR_WIDTH)
+	{
+		scroll = max(0., min(maxScroll, scroll - dy));
+	}
+	else
+	{
+		// Scroll sidebar
+		categoryScroll = max(0., min(maxCategoryScroll, categoryScroll - dy));
+	}
```

Comment:
[Style guide](https://endless-sky.github.io/styleguide/styleguide.xml):
> Single-line loop bodies: You do not need to use braces if a loop or conditional body is a single expression.
> Parentheses: Do not put a space before or after the parentheses for functions and control statements.
```suggestion
	if((hoverPoint.X() - Screen::Left()) > SIDEBAR_WIDTH)
		scroll = max(0., min(maxScroll, scroll - dy));
	else
		categoryScroll = max(0., min(maxCategoryScroll, categoryScroll - dy));
```